web/client/themes/default/less/toc.less

```diff
@@ -76,6 +75,15 @@
 
         .btn-group {
             margin: (@square-btn-size - @square-btn-medium-size)/2 0 0 0;
+
+            .btn-group {
+                margin-top: 0px;
+
+                .dropdown-toggle {
+                    width: @square-btn-medium-size;
+                    height: @square-btn-medium-size;
+                }
+            }
```

Comment:
As discussed with @allyoucanmap  Remove this 
```suggestion
```

in favor of this to put in `square-button.less`
```
.dropdown.btn-group > .square-button-md+button.dropdown-toggle {
    height: @square-btn-medium-size;
    padding: 2px;
}
```